diff --git a/buildscripts/flavors/default.sh b/buildscripts/flavors/default.sh
index 744bb61..1c5544a 100644
--- a/buildscripts/flavors/default.sh
+++ b/buildscripts/flavors/default.sh
@@ -136,0 +137,9 @@ cpuflags=
+	--enable-decoder=pgssub \
+	--enable-decoder=xsub \
+	--enable-decoder=ccaption \
+	--enable-decoder=jacosub \
+	--enable-decoder=microdvd \
+	--enable-decoder=mpl2 \
+	--enable-decoder=pjs \
+	--enable-decoder=realtext \
+	--enable-decoder=sami \
@@ -224,0 +234,4 @@ cpuflags=
+	--enable-filter=astats \
+	--enable-filter=aspectralstats \
+	--enable-filter=ametadata \
+	--enable-filter=amovie \
